safe/impact_functions/inundation/tsunami_population_evacuation_raster/metadata_definitions.py

```diff
@@ -104,7 +104,15 @@ def as_dict():
             'limitations': [tr(
                 'The default threshold of 0.7 meter was selected based on '
                 'consensus, not hard evidence.')],
-            'citations': [],
+            'citations': [
+                tr('Papadopoulos, Gerassimos A., and Fumihiko Imamura. '
+                   '"A proposal for a new tsunami intensity scale." '
+                   'ITS 2001 proceedings, no. 5-1, pp. 569-577. 2001.'),
+                tr('Hamza Latief. pers com. Default impact threshold for tsunami'
```

Comment:
Hi @Charlotte-Morgan 
Please put space in the end of the line if it's still in one text. It will be `tsunamiimpact` in the application without the space.